packages/story-editor/src/components/panels/design/textStyle/stylePresets/styleGroup.js

```diff
@@ -55,7 +59,7 @@ function StyleGroup({ styles, isEditMode, handleClick }) {
   return (
     <Group ref={groupRef}>
       {styles.map((style, i) => (
-        <ButtonWrapper key={JSON.stringify(style)}>
+        <ButtonWrapper key={JSON.stringify(style)} width={buttonWidth}>
           <StyleItem
```

Comment:
Just now noticed there's an issue with the letter-spacing displayed in the style item preview - letter spacing is greatly exaggerated. Here it's set to 3% and only showing a slight letter-spacing on canvas, but a huge spacing in the style panel preview (compare red and blue circled text previews):

![Screen Shot 2021-11-08 at 16 07 08](https://user-images.githubusercontent.com/637548/140818315-fac7ec72-58cc-4adc-9b01-921225c7192c.png)

Reply:
Good catch, looks like it was used just as `px` in the style but as `em` in the workspace.